app/exports/structured_reasons_for_rejection_export.yml

```
@@ -177,6 +177,15 @@ custom_columns:
     description: Was the reason for rejection related to other safeguarding issues?
     example: We heard from a previous school that this person should not work with children
 
+  visa_application_sponsorship:
+    type: boolean
+    description: Did you reject the candidate because you could not sponsor their visa?
```

Since this is viewed in support, can we address it to a support user rather than a provider?
```suggestion
    description: Did the provider reject the candidate because they could not sponsor the candidate’s visa?
```